modules/post/windows/gather/ad_to_sqlite.rb

```diff
@@ -377,7 +377,7 @@ def run_sqlite_query(db, table_name, values)
   def create_sqlite_db
     begin
       obj_temp = ::Dir::Tmpname
-      filename = "#{obj_temp.tmpdir}/#{obj_temp.make_tmpname('ad_', 2)}.db"
```

Comment:
> Ruby 2.5 does not have Dir::Tmpname.make_tmpname

Reply:
Did a quick grep, and `forward_pageant` uses `make_tmpname` too - but the module was also broken beyond just the call to `make_tmpname` according to this contributor https://github.com/rapid7/metasploit-framework/issues/15046